src/mudlet.cpp

```diff
@@ -3415,6 +3418,19 @@ void mudlet::slot_check_manual_update()
     updater->manuallyCheckUpdates();
 }
 
+void mudlet::slot_updateAvailable()
+{
+    // Removes the normal click to activate "About Mudlet" action and move it
+    // to a new menu which also contains a "goto updater" option
+    disconnect(actionAbout, SIGNAL(triggered()), this, SLOT(slot_show_about_dialog()));
+    actionAbout->setIcon(QIcon(QStringLiteral(":/icons/mudlet_important.png")));
+    actionAbout->setToolTip(tr("About Mudlet - <i>Note: an <b>updated version</b> is now available!</i>"));
```

Comment:
Formatting overload!

Simply `An updated version is now available` would work.

Reply:
It is still providing the "About Mudlet" action so I wanted to keep that.  The italics/bold just gives an _nice emphasis_ I thought to the __extra__ bit...